### ISSUE TITLE
Tooltip aria-live accessibility

### ISSUE THREAD

**Issue by sawmurai** [highcharts/highcharts#19595]
#### Expected behaviour

The tooltip container (the element that the content returned by the tooltip.formatter function is wrapped into) does have an [`aria-live`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) attribute as its value changes if the data changes.

#### Actual behaviour

The tooltip container (the element that the content returned by the tooltip.formatter function is wrapped into) does not have an [`aria-live`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) attribute. 

#### Product version

10.3.3

**Comment by oysteinmoseng** [comment]
Hi @sawmurai, this is by design as we found in user testing that the format used in tooltips is often not suitable for screen reader users, and a shorter, more concise format is needed. For more information on how to configure what is being read out by each data point, see [Advanced accessibility configuration](https://www.highcharts.com/docs/accessibility/configure-the-accessibility-module).

Closing this as not planned, but feel free to comment with more info and we can reopen.
